Doc/library/contextlib.rst

```diff
@@ -238,8 +238,7 @@ Functions and classes provided:
    You can capture that output in a string by redirecting the output to an
    :class:`io.StringIO` object::
 
-        f = io.StringIO()
-        with redirect_stdout(f):
+        with redirect_stdout(io.StringIO()) as f:
```

Comment:
Is just updating the example like this sufficient, or should we also explicitly mention this in the body of the text describing the behaviour?

Reply:
I think it should be explicit. `f` could also plausibly be the previous target of `sys.stdout`, so we should make clear what it is.

Reply:
31063a4 aims to do this.